OsmAnd-java/src/main/java/net/osmand/GPXUtilities.java

```diff
@@ -496,6 +496,30 @@ public boolean hasRoute() {
 		public Copyright copyright = null;
 		public Bounds bounds = null;
 
+		public Metadata() { }
+
+		public Metadata(Metadata source) {
```

Comment:
Probably we will need refactor GPS Filter task at the 4.1 branch. Since changes of GPXUtilities.java affects GPS Filter, would be good to make them via separate commit.